brian2tools/mdexport/mdexporter.py

```diff
@@ -90,9 +95,26 @@ def build(self, direct_call=True, debug=False, expander=None,
             print(self.md_text)
         elif self.filename:
             # start writing markdown text in file
-            md_file = open(self.filename + '.md', "w")
+            source_file = self.filename + ".md"
+            md_file = open(source_file, "w")
             md_file.write(self.md_text)
             md_file.close()
+            
+            # Check if Pandoc is installed
+            try:
+                subprocess.check_call(["pandoc"])
+                formats_extensions = {'latex':'.tex', 'html':'.html', 'pdf':'.pdf'}
+                if format == "all":
+                    formats = ['latex', 'html', 'pdf']
+                else:
+                    formats = format
+                for format_name in formats:
+                    filename = self.filename + formats_extensions[format_name]
+                    subprocess.run(["pandoc", "--from", "markdown", "--to", format_name, "-o", filename, source_file])
+                    print("Conversion complete! Files generated:", filename)   
+            except subprocess.CalledProcessError:
```

Comment:
This except catches only the missing installation of `pandoc`, any error that happens when running `pandoc` will be swallowed.